### PR TITLE
Update ManagedNetwork

### PR DESCRIPTION
**Pull request by yanfa317**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by yanfa317** [comment]
1, Restructure tags from list of string to single object.
2, reframe the space in managedNetwork.json

**Comment by yanfa317** [comment]
It is only be internal tested in limited area, and it is not released to outside cusomters.

**Comment by yanfa317** [comment]
@azuresdkci This patch is used to upate structure of Tags in ManagedNetwork. I verify it.

File: specification/managednetwork/resource-manager/Microsoft.ManagedNetwork/preview/2019-06-01-preview/managedNetwork.json [hunk 1]
```diff
@@ -1,1559 +1,1559 @@
 {
-  "swagger": "2.0",
```
Comment (nschonni) [inline specification/managednetwork/resource-manager/Microsoft.ManagedNetwork/preview/2019-06-01-preview/managedNetwork.json]: If you change this back to 2 spaces, the diff will be much smaller

**Review by tjprescott — CHANGES_REQUESTED** [review]
Please revert the indent changes back to 2 spaces instead of 4. It's unclear if these changes are merely fixes or are adding new things.

**Review by tjprescott — APPROVED** [review]

**Pull request merged**